### DISCUSSION TITLE
电脑传到手机的文件，在手机里没有 的问题终于靠自己解决，分享正确方法

### DISCUSSION THREAD

**Discussion by ywgdjryf** [ismartcoding/plain-app#160] [General]
看到不止一个人问过我也想问的问题：电脑传到手机的文件，存储在手机哪个目录？#107 
作者含糊的就给出一句“是的，可以在 Files -> PlainApp这个菜单查看文件。”，极其不负责任! 困扰了我几个月后最终自己解决了，正确的做法应该是：在电脑浏览器端打开首页->存储->文件，找到一个文件夹右键上传文件，再用手机打开这个文件夹。
![image](https://github.com/ismartcoding/plain-app/assets/149993388/f34d3e9c-406f-4739-918c-ba92ca59247a)

**Comment by ismartcoding** [comment]
谢谢你的分享，不过人家问的是聊天窗口发送后保存的文件在哪里。这是一个私有目录，看截图箭头这个。文件管理器里面是随便一个文件夹都能上传，本质上是因为这个就是你的手机目录。
![image](https://github.com/ismartcoding/plain-app/assets/92670034/88c2952b-436c-4b44-8bc1-ea42debc5d67)
